frontend/src/js-admin/components/OrderList.tsx

```diff
@@ -26,11 +31,17 @@ export default class OrderList extends React.Component<IOrderListProps, IOrderLi
     $.getJSON('admin-api/orders/', data, (resp: IAdminOrderListItem[]) => {
       this.setState({orders: resp});
     });
+    $.getJSON('api/shows/'+this.props.show_id, (resp: IShow) => {
+      this.setState({show: resp});
+    });
   }
 
   render() {
     return (
       <div>
+        <h2>
+          {this.state.show ? "Tilaukset - "+require('./Home.tsx').ISOToDate(this.state.show.time)+" "+this.state.show.title : ""}
```

Comment:
also, importing/`require`ing should happen in the beginning of the file and not inline